src/state/persist.go

```diff
@@ -0,0 +1,149 @@
+package state
+
+import (
+	"fmt"
+	"keboola-as-code/src/local"
+	"keboola-as-code/src/manifest"
+	"keboola-as-code/src/model"
+	"keboola-as-code/src/remote"
+	"keboola-as-code/src/utils"
+	"path/filepath"
+)
+
+// Persist created/deleted object from the filesystem
+func (s *State) Persist() (newPersisted []ObjectState, err error) {
+	s.localErrors = &utils.Error{}
+	s.newPersisted = make([]ObjectState, 0)
+
+	s.persistNewConfigs()
```

Comment:
Najprv sa najdu vsetky nove konfiguracie a potom vsetky nove rows -> uzivatel moze vytvorit v jednom kroku novu konfiguraciu spolu s rows a bude to fungovat.

Musi sa to robit takto, pretoze, ked sa pridava `row`, tak config uz musi existovat.